platforms/documentation/docs/src/docs/userguide/authoring-builds/basics/plugins.adoc

```diff
@@ -202,7 +202,7 @@ Core binary plugins provided as part of the Gradle distribution are automaticall
 The <<#sec:plugins_block,plugins DSL>> is recommended to resolve and apply plugins in one step.
 
 [[sec:binary_plugin_locations]]
-== Resolving plugins
+== Applying plugins
```

Comment:
🤔  I noticed that the first couple snippets this section is Groovy specific. Since we are making changes here, let's multi-DSL this snippet as well.

I mean this one:
```
plugins {
    id «plugin name»
}
```

And the follow up with the version.

---

Looking at them now, it seems there is little value in having the pseudo-code samples. They are simple enough, users would be able to extrapolate from the concrete samples that we, in turn, can make testable.

Reply:
I left the pseudo snippets as-is but I added examples below in both languages.